packages/web/public/static/locales/en/j-editor.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typewriterSounds": "Typewriter Sounds"
```

Comment:
@harisont just thought I'd point this extra translation string out to you, as it will be missing from the files on your branch ✌🏼🤓